bridge/svix-bridge/src/webhook_receiver/mod.rs

```diff
@@ -136,7 +135,8 @@ async fn parse_payload(
             transform(input, xform.source().clone(), transformer_tx).await
         }
         // Keep the original payload as-is if there's no transformation specified.
-        // The as_json() only gets us to `Value`, so we also need a `from_value` call to marshal into a Map type.
+        // The as_json() only gets us to `Value`, so we also need a `from_value` call to marshal
+        // into a [`ForwardRequest`] type.
         None => serde_json::from_value(payload.as_json().map_err(|_| {
             tracing::error!("Unable to parse request body as json");
             http::StatusCode::BAD_REQUEST
```

Comment:
An open question: should the default "as-is" behavior take the entire body and shove it into a `payload` field, or literally treat it as-is like it does now?

I suspect auto-wrapping is the only way we'd see folks use this without requiring a transformation.

Reply:
Yeah, take it as is and just put it in a payload field. The idea is to give people the ability to configure, but this configuration doesn't make sense in the case of webhooks. Probably only when reading from internal queues (the same way we let them control the app-id from a queue, but not from a webhook).

Reply:
Dang. Too sleepy -- I misunderstood your comment. Following-up with another PR to auto-wrap the body with `{"payload": ...}` when the transformation isn't used.

That makes sense. The body _is the payload_, after all.

Reply:
Fixed via https://github.com/svix/svix-webhooks/pull/985